Add ability to set limits at service level.

zsuzhengdu opened the pull request:
PR for https://github.com/signalfx/maestro-ng/issues/170

mpetazzoni commented:
Thanks for the contribution! Can you also update the documentation on this (and maybe provide a unit test) ?

zsuzhengdu commented:
@mpetazzoni  

I have updated doc and added 2 unit tests of limits option at service scope. Would like to hear your insightful feedback at your most convenient time. Thanks a lot!
